pkg/controllers/jobset_controller.go

```diff
@@ -266,6 +268,9 @@ func (r *JobSetReconciler) calculateReplicatedJobStatuses(ctx context.Context, j
 				log.Error(nil, fmt.Sprintf("job %s missing ReplicatedJobName label", job.Name))
 			}
 		}
+		if job.Status.Active > 0 {
+			replicatedJobsReady[job.Labels[jobset.ReplicatedJobNameKey]]["active"]++
+		}
```

Comment:
we need to check if the label exists, we do that in 265 for the ready case, can you pls move that `if` out to the beginning of the function:

```
if job.Labels != nil && job.Labels[jobset.ReplicatedJobNameKey] != "" {
  log.Error(nil, fmt.Sprintf("job %s missing ReplicatedJobName label, can't update status", job.Name))
  return
}
```